etc/bin/live-graph-movie.sh

```diff
@@ -1,127 +0,0 @@
-#!/usr/bin/env bash
-
-# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
-# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-set -eu
-
-usage() {
-    cat <<EOF
-
-USAGE: $0 <WORKFLOW>
-
-Convert frame-*.dot files in the workflow share directory (as generated by the
-old gcylc graph view if the corresponding menu option is selected) to a
-frames.webm movie, which can be played natively in the Firefox browser.
-
-Safe to re-run with different frame size and rate.
-EOF
-}
-
-# Requirements:
-#  * dot (graphviz)
-#  * convert (imagemagick
-#  * png2yuv (mjpegtools)
-#  * vpxenc (vpx-tools)
-# Documentation:
-#  * http://wiki.webmproject.org/howtos/convert-png-frames-to-webm-video
-#  * http://www.webmproject.org/docs/encoder-parameters/
-
-if [[ $1 == "-h" || $1 == "--help" || $1 == "help" ]]; then
-    usage
-    exit 0
-fi
-
-WORKFLOW=$1
-if ! cylc pr --fail $WORKFLOW >/dev/null; then
```

Comment:
Broken script relying on the removed `cylc pr` command and GCylc frames output. No one has noticed it's broken so I'm guessing it's safe to remove.